### DISCUSSION TITLE
Diagnostics aren't showing for standalone files

### DISCUSSION THREAD

**Discussion by h3xOo** [mrcjkb/rustaceanvim#488] [Q&A]
I've tried with `minimal.lua` and my personal config: on both `:checkhealth rustaceanvim` looks fine, but in `:RustLsp logFile` I get 
```
ERROR FetchWorkspaceError
rust-analyzer failed to discover workspace
```

**Comment by mrcjkb** [comment]
Hey :wave: 

I can't help you with this little information. And I cannot reproduce this with the minimal config.

Please follow the troubleshooting guide and then file [a bug report](https://github.com/mrcjkb/rustaceanvim/issues/new?assignees=&labels=bug&projects=&template=bug_report.yml) with as many details to reproduce as possible.
